src/extension/utils/ynab.ts

```diff
@@ -9,6 +9,10 @@ export function getEntityManager() {
   return ynab.YNABSharedLib.defaultInstance.entityManager;
 }
 
+export function getTransactionById(id: string) {
+  return getEntityManager().getTransactionById(id);
+}
+
```

Comment:
do you mind just calling `getEntityManager().getTransactionById(id)`  directly in the above changes instead of adding this util

Reply:
Yeah, works for me! Pushed that change :)